### PR TITLE
Add support for x86_64-unknown-linux-gnux32

### PR DESCRIPTION
**Pull request by bgermann**

### CODE REVIEW

**Comment by Bobo1239** [comment]
Just as a note: If https://github.com/rust-embedded/cross/pull/165 gets merged before this, we should change the CMake version to 3.5.1.

File: docker/x86_64-unknown-linux-gnux32/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    libc6-dev \
+    make \
+    pkg-config
+
+COPY xargo.sh /
+RUN bash /xargo.sh
+
+COPY cmake.sh /
+RUN apt-get purge --auto-remove -y cmake && \
+    bash /cmake.sh 2.8.11
```
Comment (Dylan-DPC) [inline docker/x86_64-unknown-linux-gnux32/Dockerfile]: we need to change cmake to 3.5.1

**Comment by Dylan-DPC** [comment]
bors: try

**Comment by jamesmunns** [comment]
@dylan-dpc I think it was missing a GitHub review?

**Comment by Dylan-DPC** [comment]
@jamesmunns yes, I'm aware. The thing was I was supposed to r+ another PR again which was failing due to timeout issues first and then try this one more time before review + r+. But i thought it was this PR and ended up r+ twice without a review :smile:

**Comment by jamesmunns** [comment]
Ah, now I understand. Should I cancel this merge?

**Comment by Dylan-DPC** [comment]
@jamesmunns nah its fine.

**Comment by bgermann** [comment]
Maybe later...

**Comment by Dylan-DPC** [comment]
@bgermann uhmm I was about so settle the conflicts and merge this :smile:

**Pull request closed without merging**